From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robin Ferch <robin@ferch-mail.de>
Date: Fri, 18 Feb 2022 17:08:12 +0100
Subject: [PATCH] Add BTE branding


diff --git a/src/main/java/net/minecraft/server/Eula.java b/src/main/java/net/minecraft/server/Eula.java
index a1d5c0f8fe2adb2ee56f3217e089211ec7c61eb0..78795ee81750e183a7f2932c9a345d1870203cf4 100644
--- a/src/main/java/net/minecraft/server/Eula.java
+++ b/src/main/java/net/minecraft/server/Eula.java
@@ -64,7 +64,7 @@ public class Eula {
                 try {
                     Properties properties = new Properties();
                     properties.setProperty("eula", "false");
-                    properties.store(outputStream, "By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).\nYou also agree that tacos are tasty, and the best food in the world.");  // Paper - fix lag;
+                    properties.store(outputStream, "By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).\nYou also agree that tacos are tasty, and the best food in the world and BTE is the best project in the world.");  // Paper - fix lag;
                 } catch (Throwable var5) {
                     if (outputStream != null) {
                         try {
diff --git a/src/main/java/org/bukkit/craftbukkit/Main.java b/src/main/java/org/bukkit/craftbukkit/Main.java
index f810bebfca3932c80e20646eb0680f7718506707..0ca0708322da68123278ae656780a6be348b3383 100644
--- a/src/main/java/org/bukkit/craftbukkit/Main.java
+++ b/src/main/java/org/bukkit/craftbukkit/Main.java
@@ -289,6 +289,22 @@ public class Main {
                 }
                 // Paper end
                 System.setProperty( "library.jansi.version", "Paper" ); // Paper - set meaningless jansi version to prevent git builds from crashing on Windows
+                System.out.println("""
+                    \u001B[107;40m\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\u001B[38;5;254m@\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;017m.\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;024m,\u001B[38;5;233m \u001B[38;5;002m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;017m.\u001B[38;5;017m.\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;233m \u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;022m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;017m.\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;254m&\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;024m,\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;025m*\u001B[38;5;070m*\u001B[38;5;071m*\u001B[38;5;071m*\u001B[38;5;070m*\u001B[38;5;070m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;028m*\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;017m.\u001B[38;5;254m&\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;024m,\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;233m \u001B[38;5;071m*\u001B[38;5;071m*\u001B[38;5;071m*\u001B[38;5;065m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;233m \u001B[38;5;233m \u001B[38;5;028m*\u001B[38;5;002m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;060m*\\n'
+                        + '\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;024m,\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;025m*\u001B[38;5;233m \u001B[38;5;002m,\u001B[38;5;028m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;017m,\\n'
+                        + '\u001B[38;5;017m.\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;022m.\u001B[38;5;238m,\u001B[38;5;233m \u001B[38;5;002m,\u001B[38;5;022m.\u001B[38;5;234m.\u001B[38;5;022m.\u001B[38;5;234m.\u001B[38;5;237m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;017m.\\n'
+                        + '\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;024m*\u001B[38;5;235m.\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;002m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;004m,\u001B[38;5;017m.\u001B[38;5;017m.\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;024m,\u001B[38;5;004m,\u001B[38;5;234m \u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;022m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;017m.\u001B[38;5;060m*\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;254m&\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;233m \u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;017m,\u001B[38;5;017m.\u001B[38;5;254m&\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;017m.\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;233m \u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;017m.\u001B[38;5;017m.\u001B[38;5;017m.\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\\n'
+                        + '\u001B[38;5;254m@\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;017m.\u001B[38;5;060m/\u001B[38;5;060m/\u001B[38;5;060m/\u001B[38;5;060m/\u001B[38;5;235m.\u001B[38;5;235m.\u001B[38;5;004m,\u001B[38;5;004m,\u001B[38;5;017m.\u001B[38;5;017m.\u001B[38;5;233m \u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;023m,\u001B[38;5;017m.\u001B[38;5;017m.\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\u001B[38;5;254m&\\n'
+                        + '\u001B[0m""");
+                System.out.println("\u001B[42m\u001B[37mPowered by Paper+-");
+                System.out.println(" ");
                 System.out.println("Loading libraries, please wait...");
                 net.minecraft.server.Main.main(options);
             } catch (Throwable t) {
